main/webapp/modules/core/scripts/project.js

```diff
@@ -175,35 +176,58 @@ Refine.setTitle = function(status) {
 };
 
 Refine.reinitializeProjectData = function(f, fError) {
-  $.getJSON(
-    "command/core/get-project-metadata?" + $.param({ project: theProject.id }), null,
-    function(data) {
+  Promise.all([
+    Promise.resolve($.getJSON("command/core/get-project-metadata", { project: theProject.id }))
```

Comment:
Do we use Promise somewhere else in the OR's code?

Reply:
No idea. Search for `Promise` or `then` in your IDE to find out.